src/eko/anomalous_dimensions/__init__.py

```diff
@@ -116,6 +116,7 @@ def gamma_ns(order, mode, n, nf):
         elif mode in [10201, 10200]:
             gamma_ns_1 = nlo.gamma_nsm_1(n, nf)
         else:
+            print(mode)
```

Comment:
Don't make a separate print, if you believe this information to be relevant, put it in the error message.

Use an [f-string](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings).

Reply:
Numba doesn't allow for fstrings

Reply:
If `numba` does not support f-strings, try something else: there are several ways to manipulate a string, e.g. concatenation with `+`, and this [is supported by `numba`](https://numba.pydata.org/numba-doc/dev/reference/pysupported.html#str) ;)

In general, I highlighted a problem and proposed a solution. But if the solution I proposed is not available, that's not a good reason to dismiss the problem ^^

Reply:
the solution was https://github.com/N3PDF/eko/pull/95/commits/9b8c1310eab032542e6c42b11f0b7b64cfacdcb8